packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -341,6 +342,7 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
         }
         return [];
       },
+      getMetaFields: async () => uiSettings?.get('metaFields') || [],
```

Comment:
Why do we get them from the advanced setting? I think is better to hardcode them. Check below. I have removed the _id but if I use it in my query it doesn't fail. So we should not error out or not suggest it imho

<img width="1871" alt="image" src="https://github.com/elastic/kibana/assets/17003240/689b91e3-35fd-490c-8d18-bafba951e1b1">

Reply:
I get your point but I dont think that is correct to error out a query which is not wrong and is going to return results. I am pretty sure that there is a list we can use without retrieving this information from the ui setting.

Reply:
Here it is https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-fields.html

Reply:
I've found in the ES source code a hardcoded list of metadata fields:
* `_version`
* `_index`
* `_source`
* `_id`

Reply:
Discussed offline with the ES ES|QL team and they suggested to adopt their own hardcoded list of fields.
Changed here [6faf762](https://github.com/elastic/kibana/pull/175986/commits/6faf762f7cdeb99d15ddc3b7accebd5bbd1c8a24)